#!/bin/sh /usr/share/dpatch/dpatch-run
## 106-magic-remove-emf.dpatch by Michael Piefel <piefel@debian.org>
##
## DP: Disabled EMF detection as it not only caused false positives, which might
## DP: have been acceptible, but a segmentation fault (closes: #333502).

@DPATCH@

diff -Naurp file-4.23.orig/magic/Magdir/msdos file-4.23/magic/Magdir/msdos
--- file-4.23.orig/magic/Magdir/msdos	2007-12-28 15:14:13.000000000 +0000
+++ file-4.23/magic/Magdir/msdos	2008-01-12 16:33:00.000000000 +0000
@@ -580,17 +580,17 @@
 # syntax available by some file commands you could write:
 # 0 lelong 1
 # &40 ulelong 0x464D4520 Windows Enhanced Metafile (EMF) image data
-40	ulelong 0x464D4520	Windows Enhanced Metafile (EMF) image data
->44	ulelong x		version 0x%x.
+#40	ulelong 0x464D4520	Windows Enhanced Metafile (EMF) image data
+#>44	ulelong x		version 0x%x.
 # If the description has a length greater than zero, it exists and is 
 # found at offset (*64).
->64	ulelong >0		Description available at offset 0x%x
->>60	ulelong	>0		(length 0x%x)
+#>64	ulelong >0		Description available at offset 0x%x
+#>>60	ulelong	>0		(length 0x%x)
 # Note it would be better to print out the description, which is found 
 # as below. Unfortunately the following only prints out the first couple
 # of characters instead of all the "description length"
 # number of characters -- indicated by the ulelong at offset 60.
->>(64.l)  lestring16 >0 Description: %15.15s
+#>>(64.l)  lestring16 >0 Description: %15.15s
 
 # From: Alex Beregszaszi <alex@fsn.hu>
 0	string	COWD		VMWare3
